operations/bills.md

```diff
@@ -85,6 +85,7 @@ Extent of data to be returned. E.g. it is possible to specify that together with
   "Bills": [
     {
       "Id": "26afba60-06c3-455b-92db-0e3983be0b1d",
+      "Name": "Bill Name",
```

Comment:
Can we use a proper example here, it should be anonymized but representative of real data.

Reply:
@MikeAdamsMews thank you for your comment 👍 , any suggestions ?

Reply:
I used "Accommodation Charges" as bill name , if you found it not proper feel free to reopen it again please